content/en/news/releases/1.5.x/announcing-1.5/_index.md

```diff
@@ -15,4 +15,97 @@ We are pleased to announce the release of Istio 1.5!
 
 {{< relnote >}}
 
-TBD
+We've made it simpler to
+install and run Istio by consolidating the control plane components into a
+single binary; we've introduced a powerful and fast new extension model for
+proxy servers across the industry, and we've continued to improve usability,
+security, telemetry and traffic control.
+
```

Comment:
should we have a theme for this release?

Reply:
I don't know. We've done SO MUCH in the release across the board, it's hard to pick one. If you have an idea, I'm happy to add one in...but as is, I can only think of something _really_ generic like "improvement!"